package.json

```diff
@@ -13,6 +13,8 @@
         "rebuild": "npm run clean && npm run build",
         "test": "npm run test --ws --if-present",
         "smoketest": "npm run start --workspace=smoketests",
+        "smoketest:node": "npm run start:jest --workspace=smoketests",
```

Comment:
idea: maybe we should have just one command called smoke test and execute it for all packages that have it in package.json - the same way how we run test or build

Reply:
It cannot be this way, because we don't want to run browser smoketests on each workflow on GitHub.